env.tpl

```diff
@@ -1,10 +1,11 @@
-# Do NOT edit this file!!, edit only the customer.env
+# Do NOT edit this file\!\!, edit only the customer.env
 
 ### DO NOT EDIT BELOW THIS LINE
 
 GOOGLE_APPLICATION_CREDENTIALS=../certs/key.json
 CARTO3_ONPREM_VOLUMES_BASE_PATH=./
 
+ONPREM_GCP_PROJECT_ID=carto-tnt-${ONPREM_TENANT_ID}
```

Comment:
this should not be needed, as it comes in the `customer-package`. https://github.com/CartoDB/cloud-native/blob/main/terraform/_modules/onprem/secret-manager.tf#L67